packages/yoastseo/src/scoring/assessments/readability/ParagraphTooLongAssessment.js

```diff
@@ -125,25 +125,34 @@ export default class ParagraphTooLongAssessment extends Assessment {
 				),
 			};
 		}
+
+		/* translators: %1$s and %5$s expand to links on yoast.com, %2$s expands to the anchor end tag,
+		%3$d expands to the number of paragraphs over the recommended limit, %4$d expands to the limit. */
+		const wordFeedback = _n(
+			"%1$sParagraph length%2$s: %3$d of the paragraphs contains more than the recommended maximum number of words (%4$d). %5$sShorten your paragraphs%2$s!",
+			"%1$sParagraph length%2$s: %3$d of the paragraphs contain more than the recommended maximum number of words (%4$d). %5$sShorten your paragraphs%2$s!",
+			tooLongParagraphs.length,
+			"wordpress-seo"
+		);
+		/* translators: %1$s and %5$s expand to links on yoast.com, %2$s expands to the anchor end tag,
+		%3$d expands to the number of paragraphs over the recommended limit, %4$d expands to the limit. */
+		const characterFeedback = _n(
+			"%1$sParagraph length%2$s: %3$d of the paragraphs contains more than the recommended maximum number of characters (%4$d). %5$sShorten your paragraphs%2$s!",
+			"%1$sParagraph length%2$s: %3$d of the paragraphs contain more than the recommended maximum number of characters (%4$d). %5$sShorten your paragraphs%2$s!",
+			tooLongParagraphs.length,
+			"wordpress-seo"
+		);
```

Comment:
Maybe it's better to already wrap these strings with `sprintf()` here. I think that way it's more readable for us developers when reading the translator's comment.

Reply:
Haha, I had it like that at first 😅 So I've applied your suggestion(s)! 👍